tox.ini

```diff
@@ -148,9 +156,11 @@ ignore_outcome=true
 [testenv:flake8]
 deps=flake8==3.2.0
 commands=flake8 ddtrace
-basepython=python
+basepython=python2
 
 [flake8]
 ignore=W391,E231,E201,E202,E203,E261,E302,E128,E126,E124
 max-line-length=120
-exclude = tests
+# excluding tests and async Python3.5 files
+# TODO: make the syntax Python3.4 compatible
```

Comment:
fixed in https://github.com/DataDog/dd-trace-py/pull/192 and https://github.com/DataDog/dd-trace-py/pull/191